packages/paste-website/src/pages/components/display-heading/index.mdx

```diff
@@ -0,0 +1,228 @@
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Display Heading"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/display-heading"
+  storybookUrl="/?path=/story/components-display-heading--all-variants"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{DisplayHeading}} language="jsx">
+  {`<DisplayHeading as="h1" variant="displayHeading10" marginBottom="space0">
+  Twilio magic
+</DisplayHeading>`}
+</LivePreview>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Display Heading
+
+Display Heading should not be used interchangeably with [the Heading component](/components/heading). While Heading is for adding multiple levels of hierarchy and structuring the content of any page, Display Heading is only for use in very specific contexts.
+
+Use Display Heading for marketing, branding, or editorial purposes to highlight a small amount of text. Don't use Display Heading to outline sections of a page or form.
+
+Follow these guidelines for appropriate Display Heading usage:
+
+- Don’t use Display Heading to resize text. Instead use the `<Text>` primitive
+- Use the appropriate tag for the Display Heading. If using an `<h1>` tag, don't use more than one per page.
+- Always use sentence case, unless you're writing a product name. For more information read the [product style guide](/foundations/content/product-style-guide/#sentence-case)
+
+Here are some examples of when you might want to use a Display Heading:
+
+- Grabbing your user's attention on a home or onboarding page
+- Marketing purposes or when you need to connect with the Twilio branding
+- Displaying numbers on a dashboard layout to make them stand out
+
+If you have concerns about whether you're using a Display Heading appropriately, feel free to [start a new Github Discussion](https://github.com/twilio-labs/paste/discussions/new) to ask.
+
+#### Accessibility
+
+It is important to not skip one or more heading levels as it is common for screen readers to build an outline of the page and assist users as they navigate a page. Skipping a heading level could cause confusion as the user navigating the page may not know why a heading is missing.
+
+- Display Heading text should meet AA requirements (4.5:1) for color contrast from itself and the background color
+- Display Headings should be used thoughtfully and intentionally, including the value passed to the `as` prop.
+
+## Examples
```

Comment:
Suggestion: same as the top example, I think we should highlight making the display heading a div instead of h1.